src/MizarData/include/MizarData/ActiveFunctionTimePerFrameComparator.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_DATA_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+#define MIZAR_DATA_ACTIVE_FUNCTION_TIME_PER_FRAME_COMPARATOR_H_
+
+#include <utility>
+
+#include "ClientData/ScopeStats.h"
+#include "MizarData/BaselineOrComparison.h"
+#include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "Statistics/Gaussian.h"
+#include "Statistics/StatisticsUtils.h"
+
+namespace orbit_mizar_data {
+
+// TODO(b/236130122) it should be moved to a new library
+// Implements the statistical hypothesis testing procedure aimed to test the equality of total
```

Comment:
I think you should link to your doc describing the statistical method here.

Reply:
Can we actually add links to the internal docs?

Reply:
> I you are using the "go/" link and the link name itself is not confidential, it is fine. But in general, it's good to think about it twice.

Is that the case?

Reply:
@beckerhe WDYT?

Reply:
To the best of my knowledge go-links are fine - assuming the name itself does not contain any confidential code names.